en/docs/learn/configuring-multi-tenancy.md

```diff
@@ -0,0 +1,71 @@
+# Configuring Multi-Tenancy
+
+## Scenario
+
+Consider the following scenario. 
+
+A taxi company called "Pickup" is expanding into the food delivery business and opening a new segment called "PickupEats". Pickup now has two separate business models; one for PickupEats and one for PickupTaxis. Employees and customers will still be using the same Pickup application for both services, but Pickup wishes to manage their users in two isolated environments and have different settings/policies for each. 
+
+In order to do this, Pickup can set up two tenants that share access to the same application but are isolated, so that PickupEats and PickupTaxis can function as two separate segments. This concept is called **multi-tenancy**.  This tutorial demonstrates multi-tenancy in WSO2 Identity Server. 
+
+## Create a tenant
+
+1. [Download WSO2 Identity Server](https://wso2.com/identity-and-access-management/).
+
+2. Navigate to `<IS_HOME>/bin` and start the server by executing one of the following commands.
+
+    ``` java tab="Linux/MacOS"
+    sh wso2server.sh
+    ```
+
+    ``` java tab="Windows"
+    wso2server.bat run
+    ```
+
+3. Access the [WSO2 Identity Server Management Console] (https://localhost:9443/carbon) as an administrator (credentials: admin/admin).
+
+4. On the **Configure** tab of the Management Console, click **Add New Tenant**. 
+
+5. Enter the following tenant details.
+    1. **Domain**: pickup-eats.com
+    2. **Usage plan for the tenant** - Demo
+    3. **First Name** - Jane
+    4. **Last Name** - Doe
+    5. **Admin Username** - jane@pickup-eats.com
+    6. **Admin Password** - jane123
+    7. **Email** - janedoe@gmail.com
+    
+You have now successfully created a tenant for the PickupEats. Repeat steps 4-5 and create a tenant called "pickup-taxis.com" with different admin credentials. 
```

Comment:
the first sentence can be part of the 5th instruction and the second sentence can be the 6th sentence- to create another tenant called "", repeat the ...